tests/utils/test-data-generator.ts

```diff
@@ -766,6 +780,30 @@ export class TestDataGenerator {
     };
   }
 
+  /**
+   * Generates a EventsSubscribe message for testing.
+   */
+  public static async generateEventsSubscribe(input?: GenerateEventsSubscribeInput): Promise<GenerateEventsSubscribeOutput> {
+    const author = input?.author ?? await TestDataGenerator.generatePersona();
+    const signer = Jws.createSigner(author);
+
+    const options: EventsSubscribeOptions = {
+      filters          : input?.filters,
+      messageTimestamp : input?.messageTimestamp,
+      signer,
+    };
+    removeUndefinedProperties(options);
+
+    const eventsSubscribe = await EventsSubscribe.create(options);
+    const message = eventsSubscribe.message;
+
+    return {
+      author,
+      eventsSubscribe,
+      message
+    };
```

Comment:
For consideration: there was a time that neither `message` and `author` was in the `Message` instance generated. They are now both there so we might want to just return the `EventsSubscribe` instance for simplicity. Other interface methods in this util still on old pattern mainly because we didn't get around to the low-pri refactoring.